_docs/cardano/2017-01-04-differences.md

```diff
@@ -28,20 +29,23 @@ This document is divided into four parts:
 
 ## Time, Slots, and Synchrony
 
-In a basic model of *Ouroboros* time is divided into discrete units called
+In a basic model of the protocol time is divided into discrete units called
 *slots*. However, there are no details on how to obtain current time securely
 and with enough precision.
 
-In `cardano-sl`, current time is obtained by querying a predefined set of NTP
+In Cardano SL, current time is obtained by querying a predefined set of NTP
 servers. Specifically, each node periodically queries NTP servers and calculates
 mean of results. A node stores last margin (the difference between local time and
 global time) and last obtained global time. А node also stores last slot to ensure
 that slots are monotonic.
 
 ## Coin Tossing and Verifiable Secret Sharing
 
-As *Ouroboros* paper suggests, PVSS scheme by Schoenmakers is used in
-`cardano-sl`. One of the challenges while using a VSS scheme is associating the
+The paper suggests PVSS scheme by Schoenmakers for Cardano SL. However,
+currently Cardano SL uses "SCRAPE: Scalable Randomness Attested by
+Public Entities" PVSS scheme.
```

Comment:
Perhaps it'd be good to provide a reference to this scheme? It's something new and most of the people probably don't know it :)